package.json

```diff
@@ -39,29 +39,29 @@
   "author": "Andrea Carraro <me@andreacarraro.it>",
   "license": "ISC",
   "devDependencies": {
-    "@babel/core": "^7.11.6",
-    "@testing-library/jest-dom": "^5.11.6",
-    "@testing-library/react": "^11.2.0",
-    "@testing-library/user-event": "^12.2.2",
-    "@types/find-root": "^1.1.2",
-    "@types/jest": "^26.0.14",
-    "@types/node": "^14.11.2",
-    "@types/normalize-path": "^3.0.0",
-    "@types/react": "^16.9.56",
-    "@types/react-dom": "^16.9.9",
-    "@types/webpack": "^4.41.22",
-    "babel-jest": "^26.3.0",
-    "husky": "^4.0.0",
-    "identity-obj-proxy": "^3.0.0",
-    "jest": "^26.4.2",
-    "lint-staged": "^10.0.0",
-    "next": "^10.0.2",
-    "prettier": "^2.0.0",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "react-hook-form": "6.12.2",
-    "ts-jest": "^26.4.0",
-    "typescript": "^4.0.3"
+    "@babel/core": "7.12.10",
+    "@testing-library/jest-dom": "5.11.6",
+    "@testing-library/react": "11.2.2",
+    "@testing-library/user-event": "12.5.0",
+    "@types/find-root": "1.1.2",
+    "@types/jest": "26.0.19",
+    "@types/node": "14.14.13",
+    "@types/normalize-path": "3.0.0",
+    "@types/react": "17.0.0",
+    "@types/react-dom": "17.0.0",
+    "@types/webpack": "4.41.25",
+    "babel-jest": "26.6.3",
+    "husky": "4.3.5",
+    "identity-obj-proxy": "3.0.0",
+    "jest": "26.6.3",
+    "lint-staged": "10.5.3",
+    "next": "10.0.3",
+    "prettier": "2.2.1",
+    "react": "17.0.1",
+    "react-dom": "17.0.1",
+    "react-hook-form": "6.13.0",
+    "ts-jest": "26.4.4",
+    "typescript": "4.1.3"
```

Comment:
I'm up for Renovate. If you set it up it will pin dependencies as well being the most reliable and predictable configuration. See https://docs.renovatebot.com/dependency-pinning/

Reply:
```
"rangeStrategy": "bump" :)
```
![rangeStrategy](https://media.giphy.com/media/yr7n0u3qzO9nG/giphy.gif)